### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?

As title says.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by JuanAlmeida12 — APPROVED** [review]

File: react/__tests__/Slide.test.js [hunk 1]
```diff
@@ -29,4 +29,14 @@ describe('<Slide /> component', () => {
     const { asFragment } = renderComponent()
     expect(asFragment()).toMatchSnapshot()
   })
+
+  it('should generate with tag', () => {
+    const { container } = renderComponent({ tag: 'span' })
+    expect(container.querySelector('span')).toBeTruthy()
+  })
+
+  it('should math snapshot with tag', () => {
```
Comment (matheusps) [inline react/__tests__/Slide.test.js:44]: This test already exists

File: react/__tests__/Slide.test.js [hunk 2]
```diff
@@ -29,4 +29,14 @@ describe('<Slide /> component', () => {
     const { asFragment } = renderComponent()
     expect(asFragment()).toMatchSnapshot()
   })
+
+  it('should generate with tag', () => {
```
Comment (matheusps) [inline react/__tests__/Slide.test.js]:
```suggestion
  it('should generate with te correct tag', () => {
```

File: react/__tests__/Slide.test.js [hunk 3]
```diff
@@ -29,4 +29,14 @@ describe('<Slide /> component', () => {
...
+    const { container } = renderComponent({ tag: 'span' })
+    expect(container.querySelector('span')).toBeTruthy()
```
Comment (matheusps) [inline react/__tests__/Slide.test.js:35]: Add tests with other tags like `div`, `main` would be nice 🤓

**Review by matheusps — APPROVED** [review]
🔝

**Pull request merged**